examples/connector/python/main.py

```diff
@@ -63,86 +92,77 @@ def Update(self, request, context):
 
         # -- Send UPSERT records
         for t in range(0, 3):
-            operation = connector_sdk_pb2.Operation()
             val1 = common_pb2.ValueType()
             val1.string = "a-" + str(t)
 
             val2 = common_pb2.ValueType()
             val2.double = t * 0.234
 
             record = connector_sdk_pb2.Record()
-            record.type = common_pb2.OpType.UPSERT
+            record.type = common_pb2.UPSERT
```

Comment:
Aren't these supposed to be `RecordType`?

Reply:
Yes, it works with this as well. The Operation defined globally in the file with the type.